package.json

```diff
@@ -15,7 +15,8 @@
   "dependencies": {
     "clsx": "^2.1.1",
     "react": "^19.1.0",
-    "react-dom": "^19.1.0"
+    "react-dom": "^19.1.0",
+    "react-intl": "^7.1.11"
```

Comment:
Can you update this to have exact versions? When adding a dependency, please use `--save-exact` with Yarn.

We have kept only some major dependencies like React / TypeScript etc. to be _self updating_ with `^`, as these don't have external dependency and have a history of being generally reliable. For everything else, let's keep exact version.